src/NetworkTest/testConnectivity/errors/mapping.ts

```diff
@@ -10,7 +10,8 @@
 import { ConnectivityError } from './index';
 
 export enum FailureType {
-  ConnectToSessionError = 'api',
+  APIConnectivityError = 'api',
+  ConnectToSessionError = 'media',
```

Comment:
We need to add info about the 'api' server failure type to the main README file (along with some info on what it means).